main.py

```diff
@@ -116,7 +120,7 @@ def api_req(api_endpoint, access_token, *mehs, **kw):
             print("\n\n")
 
         print("Sleeping for 2 seconds to get around rate-limit.")
-        time.sleep(2)
+        time.sleep(1)
```

Comment:
I also found that this doesn't cause too many issues. None that are there with it at 2.